model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -609,16 +609,20 @@ message Elements {
     // represents the producer of these timers.
     string transform_id = 2;
 
+    // (Optional) The local timer family name used to identify the associated
+    // timer family specification
+    string timer_family_tag = 3;
+
     // (Optional) The local timer name used to identify the associated timer specification.
-    string timer_id = 3;
+    string timer_id = 4;
 
     // (Optional) Represents a logical byte stream of a timer. Encoded according
     // to the coder in the timer spec.
-    bytes timer = 4;
+    bytes timer = 5;
```

Comment:
This should be `timers` to represent a full stream of timers.

Reply:
The `Elements` has repeated field of `Timer`. So I think the `Timer` represents a single timer,  and `Element` represent a full stream of timers for one bundle.

Reply:
The `Timer` represents some set of encoded timers for a specific PTransform and timer family.

Its important to support many timers within a `timer` and it follows the logical data stream as described here: https://s.apache.org/beam-fn-api-send-and-receive-data